File: hello.nim
import asynchttpserver
import asyncdispatch

proc app(req: Request) {.async.} =
  await(req.respond(Http200, "Hello world"))

when isMainModule:
  const port = 8080
  echo("Check http://localhost:",port)
  var server = newAsyncHttpServer()
  asyncCheck(server.serve(Port(port), app))
  runForever()
    

File: hello.go
package main

import (
	"fmt"
	"net/http"
	"runtime"
)

func handler(w http.ResponseWriter, r *http.Request) {
	w.Write([]byte("Hello, world"))
}

func main() {
	runtime.GOMAXPROCS(runtime.NumCPU())
	port := fmt.Sprintf(":%d", 8080)
	http.HandleFunc("/", handler)
	fmt.Printf("http://localhost%s\n", port)
	http.ListenAndServe(port, nil)
}
